packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -538,7 +538,7 @@ class DebuggerController extends DisposableController
   }
 
   Future<void> setExceptionPauseMode(String mode) async {
```

Comment:
rename this method to `setIsolatePauseMode`

Reply:
Done!